ui/lib/core/addon/components/radio-card.hbs

```diff
@@ -1,16 +1,16 @@
 <label
-  for={{dasherize @value}}
+  for={{dasherize @title}}
   class="radio-card {{if (eq @value @groupValue) 'is-selected'}} {{if @disabled 'is-disabled'}}"
   ...attributes
 >
   <ToolTip @verticalPosition="above" @horizontalPosition="center" as |T|>
-    <T.Trigger tabindex="-1">
+    <T.Trigger tabindex="-1" class="radio-card-container">
       {{#if (has-block)}}
         {{yield}}
       {{else}}
         <div class="radio-card-row">
           <div>
-            <Icon @name={{@icon}} @size="24" class="has-text-grey-light" />
+            <Icon @name={{@icon}} @size="24" class={{or @iconClass "has-text-grey-light"}} />
```

Comment:
For this particular use case the icon needed to be a different color

Reply:
Can we add `@iconClass` to the `RadioCard` docs?

Reply:
This is a template only glimmer component so there is no component js for the docs to live.